keyboards/ergotaco/keymaps/default/rules.mk

```diff
@@ -0,0 +1,19 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+
+#Debug options
+VERBOSE 		 		= yes
+DEBUG_MATRIX_SCAN_RATE  = no
+DEBUG_MATRIX		 	= yes
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```

Comment:
This line is already defined at the keyboard level `rules.mk`; it's redundant here.

```suggestion
```

I'll listen if you have a compelling argument for having it at both levels, though.